data/country.csv

```diff
@@ -1,218 +1,218 @@
-Niuē,Niue,Niue,نييوي,99-Other,-19.0541612,-169.8621129,NU
-Tonga,Tonga,Tonga,تونجا,99-Other,-19.7345111,-176.4580757,TO
-Saint Kitts and Nevis,Saint-Christophe-et-Niévès,Saint Kitts and Nevis,,99-Other,17.2554073,-62.6907404,KN
-Cayman Islands,Îles Caïmans,Cayman Islands,جزر الكايمان,99-Other,19.5097189,-80.5712351,KY
-Grenada,Grenade,Grenada,,99-Other,12.1913266,-61.5899005,GD
-Belau,Palaos,Palau,,99-Other,5.485,132.9200098,PW
-Saint Vincent and the Grenadines,Saint-Vincent-et-les Grenadines,Saint Vincent and the Grenadines,,99-Other,13.0498274,-61.2875808,VC
-Kūki 'Āirani,Îles Cook,Cook Islands,جزر كوك,99-Other,-15.4374746,-161.5972899,CK
-Guernsey,Guernesey,Guernsey,غيرنزي,99-Other,49.58122,-2.5356414,GG
-Tuvalu,Tuvalu,Tuvalu,توفالو,99-Other,-8.21545,0.0,TV
-Gambia,Gambie,Gambia,غامبيا,99-Other,13.4431569,-15.4133017,GM
-Tokelau,Tokelau,Tokelau,توكلو,99-Other,-8.9885565,-171.8505629,TK
-Naoero,Nauru,Nauru,ناورو,99-Other,-0.528,166.9348582,NR
-Micronesia,États fédérés de Micronésie,Federated States of Micronesia,,99-Other,5.559,150.2299283,FM
-Jersey,Jersey,Jersey,جيرزي,99-Other,49.1663334,-2.19625,JE
-Anguilla,Anguilla,Anguilla,أنجويلا,99-Other,18.4283324,-63.175872,AI
-Timór Lorosa'e,Timor oriental,East Timor,,99-Other,-8.8269117,125.7875547,TL
-São Tomé e Príncipe,Sao Tomé-et-Principe,São Tomé and Príncipe,,99-Other,0.8561232,6.9655601,ST
-Isle of Man,Île de Man,Isle of Man,جزيرة مان,99-Other,54.1992144,-4.5680673,IM
-Belize,Belize,Belize,بليز,99-Other,17.1908648,-88.2680289,BZ
-Antigua and Barbuda,Antigua-et-Barbuda,Antigua and Barbuda,أنتيغوا وباربودا,99-Other,17.343195,-62.0007544,AG
-Gibraltar,Gibraltar,Gibraltar,جبل طارق,99-Other,36.1068963,-5.3386582,GI
-འབྲུག་ཡུལ་,Bhoutan,Bhutan,بوتان,99-Other,27.4745015,90.4358523,BT
-Sāmoa,Samoa,Samoa,ساموا,99-Other,-13.7576404,-172.1010547,WS
-British Virgin Islands,Îles Vierges britanniques,British Virgin Islands,,99-Other,18.5279556,-64.5078453,VG
-Solomon Islands,Salomon,Solomon Islands,جزر سليمان,99-Other,-9.0266399,162.8577611,SB
-Pitcairn Islands,Îles Pitcairn,Pitcairn Islands,,99-Other,-24.4981252,-127.7612601,PN
-M̧ajeļ,Îles Marshall,Marshall Islands,,99-Other,9.6235,166.4843173,MH
-Kiribati,Kiribati,Kiribati,,99-Other,-3.373,1.1522793,KI
-"Saint Helena, Ascension and Tristan da Cunha","Sainte-Hélène, Ascension et Tristan da Cunha","Saint Helena, Ascension and Tristan da Cunha","سانت هيلينا، أسينسيون وتريستان دا كونها",99-Other,-24.13,-10.0230549,SH
-Kalaallit Nunaat,Groenland,Greenland,جرينلاند,99-Other,71.6952795,-42.0769588,GL
-British Indian Ocean Territory,Territoire britannique de l'Océan Indien,British Indian Ocean Territory,إقليم المحيط الهندي البريطاني,99-Other,-6.3412369,71.8692598,IO
-Falkland Islands,Îles Malouines (Îles Falkland),Falkland Islands,جزر الفولكلاند,99-Other,-51.9579887,-59.569457,FK
-San Marino,Saint-Marin,San Marino,,99-Other,43.9428966,12.4596955,SM
+name,name:fr,name:fr_short,name:en,name:ar,importance,type,latitude,longitude,iso
```

Comment:
Why keeping type AND importance?

Reply:
type is what it is, importance is more or less similar now but permits to play a bit more. For example to avoid holes in big surfaces (name of countries not showed but there is lots of space around, which is not the case everywhere).

Reply:
Not sure a "numeric" type is really useful but ok. :)

Reply:
>  the use of "<=" in selectors didn't seem to work without it, don't know

Humm, not sure to get this :s

I'm suggesting that a numeric value for type may not be a useful info given we also have a numeric value for "prio". I feel like we could deal only with "prio".

Are we using "type" somehow now?

Reply:
Ah ok, i misunderstood.

At the moment we don't use type anymore. Just that I would like to keep it in case. Not to loose the data / info that may be interesting to use after.

But yes, we can now delete the numeric prefix value